tracing/README.md

```diff
@@ -241,23 +244,42 @@ tracing::subscriber::with_default(my_subscriber, || {
 })
 ```
 
-This approach allows trace data to be collected by multiple subscribers within
-different contexts in the program. Note that the override only applies to the
-currently executing thread; other threads will not see the change from with_default.
+Any trace events generated outside the context of a subscriber will not be collected.
 
-Any trace events generated outside the context of a
-subscriber will not be collected.
+The executable itself may use the `tracing` crate to instrument itself as well.
 
-The executable itself may use the `tracing` crate to instrument itself as
-well.
+In addition to `tracing` and `tracing-core`, the [`tokio-rs/tracing`] repository
+several additional crates designed to be used with the `tracing` ecosystem.
+This includes a collection of `Subscriber` implementations, as well as utility
+and adapter crates to assist in writing `Subscriber`s and instrumenting
+applications.
+
+In particular, the following crates are likely to be of interest:
+
+ - [`tracing-futures`] provides a compatibility layer with the `futures`
+   crate, allowing spans to be attached to `Future`s, `Stream`s, and `Executor`s.
+ - [`tracing-fmt`] provides a `Subscriber` implementation for
+   logging formatted trace data to stdout, with similar filtering and
+   formatting to the `env-logger` crate.
+ - [`tracing-log`] provides a compatibility layer with the `log` crate,
+   allowing log `Record`s to be recorded as `tracing` `Event`s within the
```

Comment:
I think the reference to "log `Record`s" is probably unhelpful here, as most people (I think) aren't used to think of `log` in terms of the `Record` type. Instead, I suspect _most_ people only think of log in terms of its logging macros.